### ISSUE TITLE
508-Defect-2 [SCREENREADER] Direct Deposit Information box keyboard navigation

### ISSUE THREAD

**Issue by noahgelman** [department-of-veterans-affairs/va.gov-team#26612]
[508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Noah

## User Story or Problem Statement

The functionality of the "Direct Deposit Information" box is not accessible and there are a number of UI issues

### Direct Deposit Information block keyboard navigation

Tabbing out of the account type radio group skips the important Direction Deposit Information content block. The user needs to be able to tab to is.

## Acceptance Criteria

- [ ] Add `tabindex="0"` to Direct Deposit Information content block

**Comment by dneel-bah** [comment]
@noahgelman Can we split this issue into 2, with one being the first 2 items and the other being the last two items?  The reason for this request is because the last two items are related to the component which is used across VA.gov forms.

**Comment by noahgelman** [comment]
@dneel-bah The ticket has been split in 2. This one and https://github.com/department-of-veterans-affairs/va.gov-team/issues/28438

**Comment by zurbergram** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/18222

**Comment by dneel-bah** [comment]
Move the part of this defect that is recommended to be a platform issue into #28438; so that this can be closed out once testing is completed.

**Comment by delli-gatti-michael** [comment]
@noahgelman The fix for adding a tab index to the "Direct deposit information" content is now on staging if you would like to review

**Comment by dneel-bah** [comment]
Closing this issue as completed, based on recent guidance from OCTO, as it has passed internal validation.
